docs/source/intro/01-install.md

```diff
@@ -48,8 +47,26 @@ There is no `.app` package currently for macOS, but can try Deluge with [Homebre
 
         pip3 install deluge
 
-[develop ppa]: https://launchpad.net/~deluge-team/+archive/ubuntu/develop/
+
+## Alternative Installs
+
+### <i class="icon-python"></i> PyPi
+
+To install from Python PyPi, Deluge requires the following system installed packages:
+
+    sudo apt install python3-pip python3-libtorrent python3-gi python3-gi-cairo gir1.2-gtk-3.0 gir1.2-appindicator3
+
+Install with pip:
+
+    pip3 install deluge
+
+### Developer PPA
+
+Deluge also has a [development ppa]
```

Comment:
Deluge actually has two maintained daily build PPA, one each for develop and master branches. The stable PPA is for the latter. You have conflated the two here so in the first case it should only be mentioned as stable and link to such: https://launchpad.net/~deluge-team/+archive/ubuntu/stable/ 

It might be good to mention the other PPA but I feel a link to another document with those details is better than easily giving users access to work-in-progress code.